src/Core/Checkout/Cart/Order/Transformer/PriceTransformer.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Cart\Order\Transformer;
+
+use Shopware\Core\Framework\DataAbstractionLayer\Pricing\Price;
+use Shopware\Core\Framework\DataAbstractionLayer\Pricing\PriceCollection;
+
+class PriceTransformer
+{
+    public static function transformCollection(PriceCollection $prices): array
```

Comment:
Any specific reason why static? This cannot be extended by an Plugin

Reply:
The transformer methods are not part of any `PriceTransformer` instance. Hence they can and should be static.
I also oriented myself to the existing `AddressTransformer` https://github.com/shopware/platform/blob/b4124f2d5a04846af4e2dc643f43fdaf5e3dbbea/src/Core/Checkout/Cart/Order/Transformer/AddressTransformer.php